src/main/webapp/app/course/manage/course-management-exercises-search.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="card search-card">
+    <div class="card-header">
+        <h5 class="mb-0">{{ 'artemisApp.course.search.cardTitle' | artemisTranslate }}</h5>
+    </div>
+    <div class="card-body">
+        <div class="input-group form-group">
+            <input
+                id="exercise-search-input"
+                type="search"
+                class="form-control"
+                [placeholder]="'artemisApp.course.search.fieldExerciseName' | artemisTranslate"
+                [(ngModel)]="exerciseNameSearch"
+                (keydown.enter)="sendUpdate()"
+            />
+            <div class="input-group-append">
+                <button id="saveFilterButton" class="btn btn-primary" type="button" (click)="sendUpdate()">{{ 'artemisApp.course.search.searchButton' | artemisTranslate }}</button>
```

Comment:
The search button is connected to the name input but affects the other search elements as well.

Please make it more intuitive by placing the search button below.